### DISCUSSION TITLE
Best way to define Registry mirror for Podman machine

### DISCUSSION THREAD

**Discussion by computeralex92** [containers/podman#14924] [Q&A] [answered]
I want to replace Docker Desktop for Mac with a open source solution like Podman or Colima.
While in Colima there is a option to define when generating an environment to use a registry mirror, I'm not 100% sure what is the best way for a Podman machine.
I know I'm able to change the registries.conf within the Podman VM, but when I want to replace it (e.g. update of the OS etc), this config would be gone.
Is there any other / better way?

**Comment by rhatdan** [comment, marked as answer]
Probably not something Podman can do a lot about.  Perhaps Podman-desktop might though.
Also you could volume mount the registries.conf.d file from your host into the VM, so that you would always get it on machine replacement.

**Reply by vrothberg** [reply]
I like the idea of volume-mounting the file/directory, see `podman machine init --help`.  Doing that with `/etc/containers/registries.conf.d/` allows for adding a number registries.conf files.

**Reply by computeralex92** [reply]
Thanks a lot for your answer, I solved it with such a mount.
